src/allotropy/parsers/unchained_labs_lunatic/unchained_labs_lunatic_structure.py

```diff
@@ -36,6 +38,9 @@ def _create_measurement(
         raise AllotropeConversionError(INCORRECT_WAVELENGTH_COLUMN_FORMAT_ERROR_MSG)
 
     measurement_identifier = random_uuid_str()
+    calculated_data.extend(
```

Comment:
python is confusing... took me a while to realize that this modifies `calculated_data` 😱

Reply:
Yeah, pass-by-reference in python isn't really encouraged/common. 

I did it here because it's the fastest way to make this work, while I figure out how to get rid of it :P